client/app/main/main.html

```diff
@@ -44,7 +44,7 @@
   <div class="row text-area">
     <div class="col-sm-7 col-md-8">
       RCOS is a group of RPI students who work on open source projects.
-      Our members work on a variety of projects, which can be seen on the <a ng-href="/projects">projects</a> page. To see the presentation schedule look <a target="_blank" href="https://docs.google.com/spreadsheets/d/1A6naU2lG5ALS7RELFpeG__QHaUjLbwXmCGIyzGJum7s/edit#gid=0" title="Presentations Link">here</a>.
+      Our members work on a variety of projects, which can be seen on the <a ng-href="/projects">projects</a> page. Presentation schedule is available at the time table on the right sidebar.
```

Comment:
I think it would be fine to leave this as is, or remove the sentence altogether.